swagger.yaml

```diff
@@ -597,22 +597,6 @@ components:
                   description: Settings related to fonts.
                   type: object
                   properties:
-                    adobeFonts:
```

Comment:
Can we keep this content as a comment so that in case we want to add it back it is there